test/system/test_tpm2_create.sh

```diff
@@ -53,7 +53,7 @@ tpm2_createprimary -Q -A p -g sha1 -G rsa -C context.out
 
 # Keep the algorithm specifiers mixed to test friendly and raw
 # values.
-for gAlg in sha1 0x0B sha384; do
+for gAlg in sha1 0x0B; do
```

Comment:
We should really just query the list of supported hash algorithms and use this. Perhaps have a script with bash routines for this so the tests that need it can just source it and use them.

Maybe add a filter to tpm2_dump_capability ./tools/tpm2_dump_capability -c algorithms -fhash

Reply:
@williamcroberts I will create a common piece there.